demos/remote_attestation/maa/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -e
+
+BLUE='\033[1;34m'
+NC='\033[0m'
+INSTANCE_DIR="occlum_instance"
+bomfile="../maa.yaml"
+json_dir=$PWD/out
+
+# parameter 1 defines the predefined Occlum json
+# parameter 2 defines the generated maa json
+function gen_maa_json() {
+    config=$1
+    maa=${2:-"maa.json"}
+
+    if [ ! -f $config ]; then
+        echo "Please provide valid Occlum json file"
+        exit -1
+    fi
+
+    rm -rf ${INSTANCE_DIR} && occlum new ${INSTANCE_DIR}
+    pushd ${INSTANCE_DIR}
+
+    rm -rf image
+    copy_bom -f $bomfile --root image --include-dir /opt/occlum/etc/template
+    cp ../$config Occlum.json
+    occlum build
+
+    echo -e "${BLUE}occlum run to generate quote in maa json format${NC}"
+    occlum run /bin/gen_maa_json
+
+    echo -e "${BLUE}Generated maa json file ${json_dir}/$maa${NC}"
+    cp maa.json ${json_dir}/$maa
```

Comment:
[nit]: probably, no need to keep `maa.json` after it is copied to its destination, i.e. the file can removed after line#33